### PR TITLE
Add click to insert for chrome and firefox

### PR DESCRIPTION
**Pull request by michyprima**

### CODE REVIEW

**Comment by johannhof** [comment]
@michyprima Hello, thanks very much for you pull request. I'll try to review it as soon as possible, just wanted to let you know you've not been forgotten. :)

There are 2 tests failing, in case you want to try and reproduce locally check out https://github.com/johannhof/emoji-helper#testing

Thanks again :+1: :+1:

**Comment by michyprima** [comment]
Thanks to you!

I'm using a build including my commit right now, and CTRL+E/Y correctly opens the emoji window. Searching works, but you need to focus the search box with tab/mouse first. is this the correct behaviour?

File: shared/src/popup.js [hunk 1]
```diff
@@ -142,6 +143,11 @@
           vendor.copyToClipboard(lastCopyValue);
           showCopyMessage("Image");
         break;
+        case "insertname":
+            event.preventDefault();
```
Comment (johannhof) [inline shared/src/popup.js]: is `event.preventdefault` necessary here? :)

File: shared/src/popup.js [hunk 2]
```diff
@@ -347,6 +356,11 @@
     whatToCopy = "unicode";
     vendor.setLocal("copy-setting", "unicode");
   });
+  
+  insertName.addEventListener("click", function () {
```
Comment (johannhof) [inline shared/src/popup.js]: please wrap this with `if(insertName){...}`, that will fix the tests

**Comment by johannhof** [comment]
@michyprima I finally got around investigating the issue, tests were failing because the insertName option is not available on all platforms, you have to wrap it in an `if`. I made two remarks in the code, please look at them and we can get this merged. :+1: :+1:

**Comment by michyprima** [comment]
@johannhof thank you! fixed. should be fine now :+1:

**Comment by johannhof** [comment]
Great! let's go! Thanks for the contribution :heart:

**Comment by michyprima** [comment]
hello,

would you mind updating the extension on the store with this additions built in? I'm looking forward to get it auto-installed wherever I login with my firefox account :)

**Comment by michyprima** [comment]
hello,

would you mind updating the extension on the store with this additions built in? I'm looking forward to get it auto-installed wherever I login with my firefox account :)

**Comment by johannhof** [comment]
yup, let's make a release

**Pull request merged**